deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/LSTM.java

```diff
@@ -83,6 +85,20 @@ public LayerMemoryReport getMemoryReport(InputType inputType) {
     @AllArgsConstructor
     public static class Builder extends AbstractLSTM.Builder<Builder> {
 
+
+        public Builder recurrentConstraints(LayerConstraint... constraints) {
```

Comment:
I was thinking, and I'm not 100% sure on this, but maybe we name everything ```constraintsX``` where X is {Weights, Recurrent, Bias} etc. That way all constraint methods show up together in the javadoc and code suggestions in IDEs.